app/policies/oidc/pending_trusted_publisher_policy.rb

```diff
@@ -6,14 +6,14 @@ def resolve
   end
 
   def show?
-    record.user == user
+    same_user?(record.user)
```

Comment:
This method feels incomplete or I guess blurring the plaining a bit. We're performing a comparison, but we're now hiding one of the operands beneath the surface, so it's hard to know what's going on the surface.

Maybe if the method were something like `is_current_user?` or something, it would help improve this. Thoughts?

Reply:
I like that method name. Will update.

Reply:
I went with `current_user?`. I think it reads a little better but if we don't like it as we continue on our work, we can change it.